sematic/ui/src/appContext.ts

```diff
@@ -0,0 +1,9 @@
+import React from "react";
```

Comment:
Why do some file have `.ts` extension and others `.tsx`?

Reply:
`.tsx` extension transforms JSX syntax, eg. `<component />`. 

If you know your file will not have JSX syntax, using `.ts` will be more efficient in terms of compilation time ( IDE intellisense included).

Otherwise, using JSX syntax in a `.ts` file will incur compile errors.